frontend/public/components/dashboard/project-dashboard/project-dashboard.tsx

```diff
@@ -23,7 +23,7 @@ const mapStateToProps = ({ UI }: RootState): ProjectDashboardReduxProps => ({
   consoleLinks: UI.get('consoleLinks'),
 });
 
-const getNamespaceDashboardConsoleLinks = (
+export const getNamespaceDashboardConsoleLinks = (
```

Comment:
We should move this to a different location, otherwise we'll be pulling the dashboards into the main bundle. Moving this back into namespace.jsx is probably OK since it's already in the main bundle.

Reply:
@rhamilto correctly pointed out that the dashboards are already in the main bundle. Disregard.